src/main/java/com/boot/cart/controller/CartController.java

```diff
@@ -32,7 +32,7 @@ public class CartController {
     @Autowired
     private CartService cartService;
     
-    @PutMapping("/addProductToCart/{email}/{productName}/{quantity}")
+    @PutMapping("/{email}/{productName}/{quantity}")
```

Comment:
I think you could send productName and quantity a RequestBody and keep just email as Path variable

Reply:
id rather set this remark with a lower priority since it would have also an greater impact on the frontend side.